app/controllers/YourContactDetailsController.scala

```diff
@@ -42,6 +44,29 @@ class YourContactDetailsController @Inject()(authenticate: IdentifierAction,
   val log: LoggerLike = Logger(this.getClass)
 
   def onPageLoad(): Action[AnyContent] = authenticate async { implicit request =>
+
+    val localSessionId: SessionId = getSessionId()
+
+    localSessionId match {
+      case a => sessionCacheConnector.getSessionId(localSessionId.value) match {
```

Comment:
Better to give some meaningful name instead of a like locSessionId or ls something